KronorComponents/Common/Preview.swift

```diff
@@ -57,7 +57,7 @@ enum Preview {
             stateMachine: machine,
             networking: networking,
             returnURL: returnURL,
-            onPaymentFailure: {},
+            onPaymentFailure: {reason in },
```

Comment:
To be consistent in formatting I would suggest adding an additional space { reason in }